oak_functions/loader/src/lookup.rs

```diff
@@ -35,14 +35,19 @@ impl LookupDataAuth {
     }
 }
 
+#[derive(Clone, Debug)]
+pub enum LookupDataSource {
```

Comment:
I think the doc comment for `lookup_data_url` in https://github.com/project-oak/oak/blob/24fef4d14e1d0153c90d9ca3d7fc52b010d41327/oak_functions/loader/src/main.rs#L55 should be updated.

Reply:
done.